src/views/virtualmachines/list/components/VirtualMachineRow/VirtualMachineRow.tsx

```diff
@@ -26,10 +29,19 @@ const VirtualMachineRow: FC<
   const vmName = getName(vm);
   const vmNamespace = getNamespace(vm);
   const vmi = getVmi(vmNamespace, vmName);
+  const status = (
+    <>
+      <StatusWithPopover vm={vm} vmi={vmi} />
+      {vm?.status?.printableStatus === printableVMStatus.Migrating && (
+        <VirtualMachineMigrationPercentage vm={vm} />
+      )}
+    </>
+  );
```

Comment:
Can't we just have it in the VirtualMachineRowLayout? Why do we need to inject from here to there?

Reply:
I wanted to avoid drilling VMI object to the row, but we can have either way